tools/cldr-code/src/main/java/org/unicode/cldr/tool/GenerateTransformCharts.java

```diff
@@ -515,9 +515,11 @@ private static boolean isLatin(String target) {
     static Comparator<String> UCA;
 
     static {
-        RuleBasedCollator UCA2 = (RuleBasedCollator) Collator.getInstance(ULocale.ROOT);
+        RuleBasedCollator UCA2 =
+                (RuleBasedCollator) Collator.getInstance(ULocale.ROOT); // freeze below
         UCA2.setNumericCollation(true);
         UCA2.setStrength(Collator.IDENTICAL);
+        UCA2 = (RuleBasedCollator) UCA2.freeze();
```

Comment:
This collator could also be added to CollatorHelper, numeric and strength IDENTICAL -- there are at least two of these